src/AzSK.ADO/Framework/Core/SVT/ADO/ADO.CommonSVTControls.ps1

```diff
@@ -10,7 +10,8 @@ class CommonSVTControls: ADOSVTBase {
     hidden [object] $repoInheritePermissions = @{};
     hidden [PSObject] $excessivePermissionBitsForRepo = @(1)
     hidden [PSObject] $excessivePermissionsForRepoBranch = $null;
-    hidden [string] $repoPermissionSetId = "2e9eb7ed-3c0a-47d4-87c1-0ffdd275fd87";
+    hidden [string] $repoPermissionSetId = "2e9eb7ed-3c0a-47d4-87c1-0ffdd275fd87";    
+    [System.Collections.Generic.List[ResourceApprovalCheck]] $ResourceApprovalChecks = @();    
```

Comment:
This needs to be added to svc conn and var grp as well other wise its erroring out